google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/Subscriber.java

```diff
@@ -205,6 +206,13 @@ public static Builder newBuilder(String subscription, MessageReceiver receiver)
     return new Builder(subscription, receiver);
   }
 
+  public static int getDeliveryAttempt(PubsubMessage message) {
+    if (!message.containsAttributes("googclient_deliveryattempt")) {
+      throw new RuntimeException("Message does not contain delivery attempt information");
```

Comment:
Should we throw expectation or return 0?

Actually from the above implementation, it seems like "googclient_deliveryattempt" will always be present (defaulting to 0 for subscriptions without dead letter topic set.

Reply:
We could return 0, that works as well (or -1 for invalid and leave a comment).

The main thing that I was trying to protect against here is a user trying to use this method with a PubsubMessage that they created themselves (that does not contain the delivery attempt attribute). I can't imagine why anyone would do this so this is probably overkill and we could just return 0 instead of throwing.  WDYT?

Reply:
Yes, I agree. Let's return 0.

Reply:
Done.